tests/ui/js/storyTellingServerUITests.js

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2019 OCAD University
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENSE.txt
+*/
+
+/* global fluid, jqUnit, sjrk */
+
+"use strict";
+
+(function ($, fluid) {
+
+    jqUnit.test("Test getParameterByName function", function () {
+        var testCases = [
+            { // test retrieval of set value from provided URL
+                parameter: "testParameter",
+                url: "testUrl?testParameter=testValue",
+                expected: "testValue"
+            },
+            { // test retrieval of empty value from provided URL
+                parameter: "testParameter",
+                url: "testUrl?testParameter=",
+                expected: ""
+            },
+            { // test retrieval of set value from falsy URL
+                parameter: "testParameter",
+                url: null,
+                expected: null
+            },
+            { // test retrieval of null value from provided URL
+                parameter: null,
+                url: "testUrl?testParameter=testValue",
+                expected: null
+            },
+            { // test retrieval of set value from page URL
+                parameter: "testParameterFromUrl",
+                url: "",
+                expected: "testValue"
+            },
+            { // test retrieval of null value from page URL
+                parameter: null,
+                url: "",
+                expected: null
+            },
+            { // test retrieval of empty value from page URL
+                parameter: "emptyTestParameterFromUrl",
+                url: "",
+                expected: ""
+            },
+            { // test retrieval of missing value from page URL
+                parameter: "testParameterNotInUrl",
+                url: "",
+                expected: null
+            }
+        ];
+
+        jqUnit.expect(testCases.length);
+
+        fluid.each(testCases, function (testCase, index) {
+            if (index === 4) {
+                sjrk.storyTelling.testUtils.setQueryString("testParameterFromUrl=testValue&emptyTestParameterFromUrl=");
+            }
+
+            var actualResult = sjrk.storyTelling.getParameterByName(testCase.parameter, testCase.url);
+            jqUnit.assertEquals("Query string parameter '" + testCase.parameter + "' is retrieved as expected", testCase.expected, actualResult);
+        });
+    });
+
+    // var browsePageOptions = {
```

Comment:
Are these commented lines are for the test meant to be dealt with at https://issues.fluidproject.org/browse/SJRK-255?

Reply:
That's correct, and the same is true of the last two elements of the test sequence

Reply:
Can you track these tests-in-progress in a separate branch so we can merge clean code into the master. Thanks.

Reply:
bump

Reply:
bump again. Please remove the commented code and track it in your own branch so we can merge the clean code. Thanks.